user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Creating_GQI_query.md

```diff
@@ -27,6 +27,10 @@ The "Queries" data item allows you to construct a query in order to tap into the
 
 1. Select an operator. This step is optional; if you do not select an operator, the data set will be returned untouched. See [Query operators](xref:Query_operators).
 
+   - From DataMiner 10.4.0 [CU13]/10.5.0 [CU1]/10.5.4 onwards<!--RN 42127-->, you can insert new operators anywhere in the sequence, including between existing operators, by clicking the "+" button.
```

Comment:
Maybe we could add another step above this, e.g. "1. Add more operators if necessary until your query is fully configured."? Right now it doesn't make much sense to mention inserting or moving operators in the sequence as only one operator has been added in the procedure.

Reply:
Done in [commit 6f8b937](https://github.com/SkylineCommunications/dataminer-docs/pull/4361/commits/6f8b9374edb0ad310ff4d8cce626446501cd0e5d)